### DISCUSSION TITLE
CORS problem

### DISCUSSION THREAD

**Discussion by zhudayong1987** [developmentseed/eoAPI#169] [Q&A]
I deployed my project with docker-compose and had cross-domain issues when I accessed the 8085 interface. When I mask cross-domain problems with my browser, a new problem arises: The response is not a valid JSON object."

#### Environment Information

Os, Python version, ...
Linux iZbp10g8hc3k94s7mikt6lZ 3.10.0-1160.102.1.el7.x86_64 #1 SMP Tue Oct 17 15:42:21 UTC 2023 x86_64 x86_64 x86_64 GNU/Linu

**Comment by emileten** [comment]
Hi @zhudayong1987 ! Thanks for raising this. So that we can help you, would you mind posting a little more information ?

In particular : 

- The exact command you ran to deploy the services, and any configuration change you applied to the repository before running that command. 
- The exact error message you encountered

Thank you !
